components/dashboard/src/components/ContextMenu.tsx

```diff
@@ -73,7 +89,10 @@ function ContextMenu(props: ContextMenuProps) {
     // Default 'children' is the three dots hamburger button.
     const children = props.children || (
         <svg
-            className="w-8 h-8 p-1 rounded-md text-gray-600 dark:text-gray-300 hover:bg-gray-200 dark:hover:bg-gray-700"
+            className={cn(
+                "w-8 h-8 p-1 rounded-md text-gray-600 dark:text-gray-300",
+                expanded ? "bg-gray-200 dark:bg-gray-700" : "hover:bg-gray-200 dark:hover:bg-gray-700",
+            )}
```

Comment:
**issue**: Probably some color mismatch but the workspaces list hover state changes color after expanding the more actions menu, right?

https://user-images.githubusercontent.com/120486/216107110-fff224de-d659-4a4b-b206-8fc1da204bf4.mov

Reply:
Yes @gtsiolis, you're right there was a color mismatch, I have fixed it with a commit.